SharedExpensesApp/app/src/main/java/com/example/sharedexpensesapp/ui/screens/AccountScreen.kt

```diff
@@ -1,8 +1,176 @@
+                            .padding(16.dp),
+                        style = TextStyle(fontSize = 30.sp),
+                        textAlign = TextAlign.Center,
+                        color = colorResource(id = R.color.main_orange),
+                    )
+                }
+
+            }
+            Text(
+                text = "Settings",
+                style = TextStyle(fontSize = 30.sp),
+                textAlign = TextAlign.Center,
+                color = colorResource(id = R.color.purple_700),
+            )
+            TextField(
+                value = name,
+                onValueChange = { name = it },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp),
+                label = { Text("Name") },
+
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next
+                ),
+                keyboardActions = KeyboardActions(
+                    onNext = { focusManager.moveFocus(FocusDirection.Down) }
+                )
+            )
+            TextField(
+                value = nickname,
+                onValueChange = { nickname = it },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp),
+                label = { Text("Nickname") },
+                visualTransformation = PasswordVisualTransformation(),
+
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Done
+                ),
+                keyboardActions = KeyboardActions(
+                    onDone = { focusManager.clearFocus() }
+                )
+            )
+
+            // Toggle Button for Notifications
+            Spacer(modifier = Modifier.height(16.dp))
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(8.dp),
+                horizontalArrangement = Arrangement.Center,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text("Receive Notifications", modifier = Modifier.alignByBaseline() ,
+                        style = TextStyle(fontSize = 30.sp),
+                    textAlign = TextAlign.Center,
+                    color = colorResource(id = R.color.purple_700),)
+                Spacer(modifier = Modifier.width(15.dp))
+                Switch(
+                    checked = notificationsEnabled,
+                    onCheckedChange = { notificationsEnabled = it },
+                    colors = SwitchDefaults.colors(
+                        checkedThumbColor = colorResource(id = R.color.purple_200),
+                        uncheckedThumbColor = colorResource(id = R.color.purple_500),
+                        checkedTrackColor = colorResource(id = R.color.purple_700),
+                        uncheckedTrackColor =colorResource(id = R.color.purple_500),
+                    )
+                )
```

Comment:
Instead of writing a comment, extract this into a composable. Much more readable.